handlers/web/categoriesWebHandler.js

```diff
@@ -26,8 +27,24 @@ CategoriesWebHandler.technologiesForCategory = function (req, res) {
             return;
         }
 
+        // groups technologies by status into the following structure: 
+        // [{ status: key, technologies: [technologies where status==key]}]
+        var technologiesInGroups = _.chain(values).groupBy('status')
+            .map(function(technologies, key) {
+                return {
+                    status: key,
+                    technologies: technologies
+                };
+            }).value();
+
         var category = cache.getCategory(cname);
-        res.render('pages/categoryRadar', {category: category, technologies: values, user: req.user});
+
+        res.render('pages/categoryRadar', {
+            category: category,
+            technologies: values, // used by radar.js
+            technologiesInGroups: technologiesInGroups,
```

Comment:
`technologiesInGroups` and  `technologies` contain the same data (structured differently). I couldn't get rid of `technologies` without reworking radar.js. Changing radar.js to use `technologiesInGroups` might take a whole workday. Is it worth it?